### ISSUE TITLE
Code get stucked internally in command- Genetic Algorithm with Multilayer Perceptron(SKLearn)

### ISSUE THREAD

**Issue by jineegoyal250794** [appherox/sapmle-site#4]
I am using Genetic algorithm to select features combining with Multilayer Perceptron
I an using Python Anaconda Spyder

python 3.6.8
Anaconda 5.3.1
Spyder 3.3.2
tensorflow-gpu 1.12.0
sklearn-genetic 0.2
Here is the code snippet

total_cols = len(dataset.columns); 
print(total_cols)
X=dataset.values[:,0:total_cols-1]
Y=dataset.values[:,total_cols-1]

def build_classifier():
    classifier = Sequential()
    classifier.add(Dense(activation="relu", input_dim=752, units=10, kernel_initializer="uniform"))
    classifier.add(Dense(activation="relu", units=10, kernel_initializer="uniform"))
    classifier.add(Dense(activation="sigmoid", units=1, kernel_initializer="uniform"))
    classifier.compile(optimizer = 'adam', loss = 'binary_crossentropy', metrics = ['accuracy'])
    return classifier
classifier = KerasClassifier(build_fn = build_classifier, batch_size = 10, epochs = 10)

selector = GeneticSelectionCV(classifier,
                                  cv=loocv,
                                  verbose=1,
                                  scoring="accuracy",
                                  max_features=20,
                                  n_population=25,
                                  crossover_proba=0.5,
                                  mutation_proba=0.2,
                                  n_generations=20,
                                  crossover_independent_proba=0.5,
                                  mutation_independent_proba=0.05,
                                  tournament_size=3,
                                  n_gen_no_change=10,
                                  caching=True,
                                  n_jobs=-1)

selector = selector.fit(X,Y)

It is getting stuck (running indefinitely) in the last command 
X contains 752 columns and 754 rows
